packages/react-native/package.json

```diff
@@ -1,13 +1,20 @@
 {
   "name": "react-native",
   "version": "1000.0.0",
-  "bin": "./cli.js",
   "description": "A framework for building native apps using React",
   "license": "MIT",
-  "repository": "github:facebook/react-native",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/facebook/react-native.git",
+    "directory": "packages/react-native"
+  },
+  "homepage": "https://reactnative.dev/",
+  "keywords": ["react", "react-native", "android", "ios", "mobile", "cross-platform", "app-framework", "mobile-development"],
```

Comment:
@cortinico 

Are we okay with these keywords?

Reply:
PS. I took them from keyword on RN gh repo page :)